ui/component/or-attribute-input/src/or-anomaly-config-input.ts

```diff
@@ -0,0 +1,358 @@
+                    <or-mwc-input style="padding-top: 10px;"  required="true" type="select" .label="${i18next.t("type")}" .options="${types.map((obj) => obj.label) || undefined}"
+                                  .value="${method.type ? types.filter((obj) => obj.value === this.anomalyDetectionConfigObject!.methods![this.selectedIndex].type).map((obj) => obj.label)[0] : ""}"
+                                  @or-mwc-input-changed="${(e: OrInputChangedEvent) => {
+                                      e.detail.value = types.filter((obj) => obj.label === e.detail.value).map((obj) => obj.value)[0]
+                                      this.updateProperty(e,"type");
+                                  }}"></or-mwc-input>
+                    ${DetectionMethodConfig(method)}
+                </div>
+                
+            `;
+        }
+        const DetectionMethodConfig = (method: AnomalyDetectionConfigurationUnion) =>{
+            switch (method.type){
+                case "change":
+                    return html`
+                        <or-mwc-input style="padding-top: 10px;" required="true" type="number" .label="${i18next.t("anomalyDetection.deviationChange")}" .value="${method.deviation}" @or-mwc-input-changed="${(e: OrInputChangedEvent) => this.updateProperty(e,"deviation")}"></or-mwc-input>
+                        <or-mwc-input style="padding-top: 10px;" required="true" type="text" .label="${i18next.t("anomalyDetection.minimumTimespan")}" .value="${method.timespan}" @or-mwc-input-changed="${(e: OrInputChangedEvent) => this.updateProperty(e,"timespan")}"></or-mwc-input>
+                    `;
+                case "global":
+                    return html`
+                        <or-mwc-input style="padding-top: 10px;" required="true" type="number" .label="${i18next.t("anomalyDetection.deviationGlobal")}" .value="${method.deviation}" @or-mwc-input-changed="${(e: OrInputChangedEvent) => this.updateProperty(e,"deviation")}"></or-mwc-input>
+                        <or-mwc-input style="padding-top: 10px;" required="true" type="text" .label="${i18next.t("anomalyDetection.minimumTimespan")}" .value="${method.timespan}" @or-mwc-input-changed="${(e: OrInputChangedEvent) => this.updateProperty(e,"timespan")}"></or-mwc-input>
+                    `;
+                case "forecast":
+                    return html`
+                        <or-mwc-input style="padding-top: 10px;" required="true" type="number" .label="${i18next.t("anomalyDetection.deviationGlobal")}" .value="${method.deviation}" @or-mwc-input-changed="${(e: OrInputChangedEvent) => this.updateProperty(e,"deviation")}"></or-mwc-input>
+                    `;
+                default:
+                    return html`
+                        
+                    `;
+            }
+        }
+        window.setTimeout(() => doLoad(this.anomalyDetectionConfigObject as AnomalyDetectionConfigObject), 0);
+        this.requestUpdate();
```

Comment:
Don't know what this exactly does, but I think you're better off with a different solution using the lifecycle.
As this piece of code gets executed on every render. `firstUpdated()` will only trigger after the 1st render, while `updated()` does it after every render. To compute values during (every) render you can use `willUpdate()`.
See Lit docs [here](https://lit.dev/docs/v2/components/lifecycle/#reactive-update-cycle).